sdks/java/core/src/main/java/org/apache/beam/sdk/util/WindowedValue.java

```diff
@@ -187,13 +182,6 @@ public T getValue() {
     return windowedValues.build();
   }
 
-  /**
-   * Returns the pane of this {@code WindowedValue} in its window.
-   */
-  public PaneInfo getPane() {
-    return pane;
-  }
-
   @Override
   public abstract boolean equals(Object o);
```

Comment:
Allowing other implementations of `WindowedValue` will mean that the existing equals methods won't behave as expected (given the specialization of window cardinality & globality); We should probably either document that or implement equality via method calls.